test/js/node/fs/fs.test.ts

```diff
@@ -1044,6 +1044,19 @@ describe("readSync", () => {
     }
     closeSync(fd);
   });
+
+  it("works with offset + length passed but not position", () => {
+    const fd = openSync(import.meta.dir + "/readFileSync.txt", "r");
+    const four = new Uint8Array(4);
+    {
+      const count = readSync(fd, four, 0, 4);
```

Comment:
this tests length, but not offset. can this test both?
```suggestion
      const count = readSync(fd, four, 1, 5);
```